### PR TITLE
Travis: use Ubuntu Xenial

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +35,7 @@ matrix:
   allow_failures:
   - env: UVLOOP="y"
   - python: "pypy3.5-5.8.0"
```
Comment (webknjaz) [inline .travis.yml]: This could also be updated to 6.0.0

Comment (blueyed) [reply .travis.yml]: I wasn't sure which versions are available etc.
Let's do it in another PR then?

Comment (webknjaz) [reply .travis.yml]: Sure. It's poorly advertised which PyPy versions are available under Travis CI...
Still, this PR seems to fail with current changes

Comment (webknjaz) [reply .travis.yml]: Oh, it was flaky

**Pull request merged**